tsconfig.json

```diff
@@ -13,7 +13,7 @@
         "experimentalDecorators": true,
         "importHelpers": true,
         "target": "es2015",
-        "types": ["node"],
+        "types": ["jasmine", "jest", "mocha", "node"],
```

Comment:
I think adding jasmine alone is enough, no need for jest and mocha. It's going to be confusing for maintainers

Reply:
When I was testing I get errors if I don't add jest and mocha here. Can you provide a fix if you know how to do it?

Reply:
Can we transfer the config we have in our old Angular components which works btw to here ?

Reply:
you can try but it's more or less the same here. Might be differences in how the config has to be setup because this is Angular 8 and the old one was Angular 7